src/qt/forms/openuridialog.ui

```diff
@@ -30,6 +30,23 @@
        </property>
       </widget>
      </item>
+     <item>
+      <widget class="QPushButton" name="pasteButton">
+       <property name="toolTip">
+        <string>Paste address from clipboard</string>
+       </property>
+       <property name="text">
+        <string/>
+       </property>
+       <property name="icon">
+        <iconset resource="../bitcoin.qrc">
+         <normaloff>:/icons/editpaste</normaloff>:/icons/editpaste</iconset>
+        </property>
+        <property name="shortcut">
+         <string>Alt+P</string>
```

Comment:
Why this shortcut instead of traditional Ctrl+V?

Reply:
To make it coherent with the other paste buttons

Reply:
Is it worth to present the shortcut to a user somehow? In `placeholderText`?

Reply:
Honestly I don't see value in the shortcut, I'd remove it from the others and make sure the line edit has focus.